packages/react-focus/README.md

```diff
@@ -0,0 +1,11 @@
+# @fluentui/react-focus
+
+**React Focus components for [Fluent UI](https://fluent.microsoft.com/)**
```

Comment:
Can we update the create-package template to do this (if it is what we want for other packages too)?